add_events.py

```diff
@@ -112,22 +108,22 @@ def create_calendar():
                 replaceHour += int(startHour)
             event = {}
             # Currently there are no labs in `subjects.json`
-            # if (data[day][time][0] in subjects.keys()):
```

Comment:
Could you remove comments that are irrelevant now? Actually even I never noticed these. 
There might be other such commented lines too, so please do check